crocket/src/main/java/com/crocket/model/entity/CollisionHandler.java

```diff
@@ -30,50 +31,60 @@ static boolean intersect(Hitbox hitbox1, Hitbox hitbox2) {
         int h2Width = hitbox2.getWidth();
         int h2Height = hitbox2.getHeight();
 
-        boolean xMax1IsGreaterThanxPos2 = h1X + h1Width > h2X;
+        return (h1X + h1Width > h2X &&
```

Comment:
Does this work better than the previous solution?

Reply:
Pontus mentioned that the code behaves oddly with small hitboxes, such as those found in post hitboxes within the hoop.